src/test/regress/expected/gporca_optimizer.out

```diff
@@ -12677,8 +12677,8 @@ where out.b in (select coalesce(tcorr2.a, 99)
    ->  Gather Motion 3:1  (slice1; segments: 3)  (cost=0.00..431.00 rows=1 width=8)
          ->  Seq Scan on tcorr1  (cost=0.00..431.00 rows=1 width=8)
    SubPlan 1  (slice0)
-     ->  Result  (cost=0.00..1324032.56 rows=1 width=4)
-           ->  Nested Loop Left Join  (cost=0.00..1324032.56 rows=1 width=4)
+     ->  Result  (cost=0.00..1324032.56 rows=2 width=4)
+           ->  Nested Loop Left Join  (cost=0.00..1324032.56 rows=2 width=4)
```

Comment:
Why it's OK to see 2 rows here?

Reply:
Yes, this is correct behavior.  For both of this plan nodes `plan_rows` is `1.2`. Before this PR for the plan generated orca `scaleFactor` was equal to 3 (the number of segments) only because the flow for these nodes is `NULL`. Now `scaleFactor = 1`. 

It is also worth mentioning another tests. In the second test, where the rows are changed to 3, although if you run `explain analyze` will be 1, the problem in the orca. The problem is in the wrong estimation of the number of tuples returned by the orca. There is no problem in the output.

Reply:
@t1mursadykov nope, it is an **incorrect** behaviour. You make a local NL join - a single row in a left node and a single row in the right node. As a result, you should get zero or a single row as an output, not two rows.